pages/configuration/web_interface.rst

```diff
@@ -192,6 +192,27 @@ Apache httpd 2.x
 
     </VirtualHost>
 
+**REST API and Web Interface on one port (using HTTPS/SSL)**::
+
+    <VirtualHost *:443>
+        ServerName graylog.example.org
+        ProxyRequests Off
+        SSLEngine on
+        # <- your SSL Settings here!
+
+        <Proxy *>
+            Order deny,allow
+            Allow from all
+        </Proxy>
+
+        <Location />
```

Comment:
Is wrapping the `Proxy*` directives inside the `Location` block still required, given that it points to the root resource (`/`)?